pkg/k8s/apis/cilium.io/const.go

```diff
@@ -64,6 +64,11 @@ const (
 	CiliumK8sAnnotationPrefix = "cilium.io/"
 	// CiliumIdentityAnnotationDeprecated is the previous annotation key used to map to an endpoint's security identity.
 	CiliumIdentityAnnotationDeprecated = "cilium-identity"
+
+	// AgentNotReadyNodeTaint is removed from nodes' taints which prevents pods
```

Comment:
Comment wording nit:

"AgentNotReadyNodeTaint is a node taint which prevents pods from being scheduled. Once cilium is setup it is removed from the node. Mostly used in cloud providers to prevent existing CNI plugins from managing pods"